layouts/partials/nav.html

```diff
@@ -8,7 +8,7 @@
           <span aria-hidden="true"></span>
           <span aria-hidden="true"></span>
         </a>
-        <div class="navbar-menu" id="navMenu">
```

Comment:
Instead of setting a style, you can just add the bulma class `has-text-centered`.

Reply:
Nope, doesn't cut it with just text-alignment, it really needs to be justify-content here. Couldn't find any bulma class which worked.

Reply:
Your correct. But I still think inline style isn't the right way to go. You you add a class to https://github.com/vickylai/hugo-theme-introduction/blob/master/assets/sass/_base.sass#L123?

Reply:
Your correct. But I still think inline style isn't the right way to go. You you add a class to https://github.com/vickylai/hugo-theme-introduction/blob/master/assets/sass/_base.sass#L123?

Reply:
Your correct. But I still think inline style isn't the right way to go. You you add a class to https://github.com/vickylai/hugo-theme-introduction/blob/master/assets/sass/_base.sass#L123?

Reply:
Your correct. But I still think inline style isn't the right way to go. You you add a class to https://github.com/vickylai/hugo-theme-introduction/blob/master/assets/sass/_base.sass#L123?

Reply:
Your correct. But I still think inline style isn't the right way to go. You you add a class to https://github.com/vickylai/hugo-theme-introduction/blob/master/assets/sass/_base.sass#L123?

Reply:
Your correct. But I still think inline style isn't the right way to go. You you add a class to https://github.com/vickylai/hugo-theme-introduction/blob/master/assets/sass/_base.sass#L123?

Reply:
Your correct. But I still think inline style isn't the right way to go. You you add a class to https://github.com/vickylai/hugo-theme-introduction/blob/master/assets/sass/_base.sass#L123?

Reply:
Your correct. But I still think inline style isn't the right way to go. You you add a class to https://github.com/vickylai/hugo-theme-introduction/blob/master/assets/sass/_base.sass#L123?

Reply:
Your correct. But I still think inline style isn't the right way to go. You you add a class to https://github.com/vickylai/hugo-theme-introduction/blob/master/assets/sass/_base.sass#L123?

Reply:
Your correct. But I still think inline style isn't the right way to go. You you add a class to https://github.com/vickylai/hugo-theme-introduction/blob/master/assets/sass/_base.sass#L123?

Reply:
Your correct. But I still think inline style isn't the right way to go. You you add a class to https://github.com/vickylai/hugo-theme-introduction/blob/master/assets/sass/_base.sass#L123?

Reply:
Your correct. But I still think inline style isn't the right way to go. You you add a class to https://github.com/vickylai/hugo-theme-introduction/blob/master/assets/sass/_base.sass#L123?

Reply:
Your correct. But I still think inline style isn't the right way to go. You you add a class to https://github.com/vickylai/hugo-theme-introduction/blob/master/assets/sass/_base.sass#L123?

Reply:
Alright, this part should be solved now.